src/notation/view/widgets/editstringdata.cpp

```diff
@@ -49,6 +49,8 @@ EditStringData::EditStringData(QWidget* parent, std::vector<mu::engraving::instr
     _strings = strings;
     stringList->setHorizontalHeaderLabels({ qtrc("notation/editstringdata", "Always open"),
                                             qtrc("notation/editstringdata", "Pitch") });
+    stringList->horizontalHeaderItem(0)->setToolTip(qtrc("notation/editstringdata",
+                                                         "<b>Always open</b><br>On tablature staves, fret positions other than “0” cannot be entered on strings marked ‘always open’. Useful for instruments with strings that are not on the fretboard, such as some lutes, theorbo, torban."));
```

Comment:
Let's use single quotes everywhere (`‘0’`) and shorten the ending to just "such as the theorbo".

Reply:
You can also store this string in a `QString` variable and then reference it below so you only have to write it once.